### PR TITLE
Resolve directories earlier so Symfony endpoint change works

### PR DESCRIPTION
**Pull request by andytson-inviqa**

### CODE REVIEW

**Comment by sroze** [comment]
This is to fix #405 by @TerjeBr, right?

**Comment by andytson-inviqa** [comment]
@sroze yes, just testing it now

File: php/shared/usr/local/share/env/55-stack [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-export WEB_DIRECTORY=${WEB_DIRECTORY:-web}
```
Comment (TerjeBr) [inline php/shared/usr/local/share/env/55-stack]: Why did you move `WEB_DIRECTORY` earlier? It seems kind of unrelated.

**Comment by andytson-inviqa** [comment]
All looks ok now on Symfony 2.8/3.3(no flex) and 3.3(flex). endpoints resolve correctly, and loading the default page loads as expected for each

File: php/shared/usr/local/share/env/55-stack (hunk 1 above)
Comment (andytson-inviqa) [reply php/shared/usr/local/share/env/55-stack]: @TerjeBr it was because Flex moves the web directory to "public", the defaults when it was in 45-framework needed to be before the *-stack, so left 55-stack as the only option. Since 45-framework is now 37-framework and before 40-stack, it was able to be moved back to where it was before
